tasks/build/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = function (grunt) {
       'clean:build',
       'clean:target',
       '_build:downloadNodeBuilds:start',
+      '_build:downloadNodeBuilds:finish',
```

Comment:
The reason why we put downloadNodeBuilds:finish further down the build task is to allow other build tasks to occur while the binaries are downloading.  I'm not opposed to blocking entirely on the downloadNodeBuilds task if that's necessary, but if it is, we should roll it all into a single task.